dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/configuration/ImportVariantsStepConfigurationTest.java

```diff
@@ -76,16 +81,38 @@ public void loadVariants() throws Exception {
         for (DBObject dbObject : dbObjects) {
             BasicDBList ids = (BasicDBList) dbObject.get("dbsnpIds");
             totalSnps += ids.stream().filter(o -> ((String) o).startsWith("rs")).count();
-            totalSubsnps += ids.stream().filter(o -> ((String)o).startsWith("ss")).count();
+            totalSubsnps += ids.stream().filter(o -> ((String) o).startsWith("ss")).count();
         }
 
-        assertEquals(8, dbObjects.size());
-        assertEquals(8, totalSnps);
+        assertEquals(9, dbObjects.size());
+        assertEquals(9, totalSnps);
         assertEquals(12, totalSubsnps);
+
+        assertSnpCoordinatesAreCorrect();
+        assertInsertionCoordinatesAreCorrect();
```

Comment:
Single method that could receive the coordinates as parameters? It would make more explicit which coordinates are being tested, right now it could seem like all the SNPs and insertions are.

Reply:
I tried to make that more clear without decreasing readability